clients/extension/scripts/settings-controller.js

```diff
@@ -4,6 +4,14 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+const BACKENDS = [{
+  id: 'psi',
+  title: 'pagespeed.web.dev',
+}, {
+  id: 'viewer',
+  title: 'googlechrome.github.io',
```

Comment:
I still think it makes sense to use more descriptive titles. I guess I can understand the `pagespeed.web.dev` title as PSI but it's not entirely clear that `googlechrome.github.io` refers to the Lighthouse viewer.